addons/binding/org.openhab.binding.etherrain/src/main/java/org/openhab/binding/etherrain/internal/EtherRainConfiguration.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) 2010-2019 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.etherrain.internal;
+
+import java.util.Collections;
+import java.util.Set;
+
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+
+/**
+ * The {@link EtherRainConfiguration} class contains fields mapping thing configuration parameters.
+ *
+ * @author Joe Inkenbrandt - Initial contribution
+ */
+public class EtherRainConfiguration {
+
+    public static final String BINDING_ID = "etherrrain";
+
+    public static final ThingTypeUID ETHERRAIN_THING_TYPE = new ThingTypeUID(BINDING_ID, "etherrain");
+
+    public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = Collections.singleton(ETHERRAIN_THING_TYPE);
+
+    /**
+     * Hostname of the OpenSprinkler API.
+     */
+    public String hostname = null;
```

Comment:
why don't you set the default host here? Also check that the documentation refers to `host`